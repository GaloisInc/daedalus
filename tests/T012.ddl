
def x = 'a'
def $x = !x
def Main = $x
