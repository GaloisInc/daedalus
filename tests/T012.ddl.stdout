module T012
 
--- Imports:
 
--- Type defs:
 
--- Rules:
 
T012.x : uint 8 =
  'a'
 
T012.$x : ByteClass =
  !{T012.x}
 
T012.Main : Grammar (uint 8) =
  Match T012.$x
 
T012._Main : Grammar {} =
  @Match T012.$x
